library.properties

```diff
@@ -1,9 +1,9 @@
 name=Callback
-version=1.0
+version=1.1
 author=Tom Stewart <tomstewart89@hotmail.com>
-maintainer=Tom Stewart <tomstewart89@hotmail.com>
+maintainer=NachtRaveVL <nachtravevl@gmail.com>
 sentence=A simple implementation of Signals and Slots for writing more flexible callbacks
 paragraph=Similar to Boost::Signals and a handy algorithm when writing event driven programs
 category=Other
-url=https://github.com/tomstewart89/Callback
+url=https://github.com/NachtRaveVL/Callback
```

Comment:
I appreciate the work you put in on this PR, but as the original author of the library I'd prefer for the url to continue to point to my repo. I hope I'm not being too selfish here

Reply:
Oh not at all selfish! :p Go ahead and merge it into your stuff! ^_^